controllers/sessions.js

```diff
@@ -1,12 +1,20 @@
 const User = require("../models/user");
+const { body, validationResult } = require("express-validator");
 
 const SessionsController = {
   New: (req, res) => {
     res.render("sessions/new", {});
   },
 
   Create: (req, res) => {
-    console.log("trying to log in");
+    const errors = validationResult(req);
+
+    if (!errors.isEmpty()) {
+      return res.status(422).render("sessions/new", {
+        message: JSON.stringify(errors.array()[0].msg).replaceAll('"', ""),
```

Comment:
Can be changed to `message: errors.array()[0].msg,`, as per comment on previous PR.

Reply:
Resolved!